src/commands.c

```diff
@@ -348,22 +375,52 @@ RRStatus CommandSpecTableSetC(CommandSpecTable *cmd_spec_table, void *key, size_
 /* Look up the specified command in the command spec table and return the
  * CommandSpec associated with it, or NULL.
  */
-const CommandSpec *CommandSpecTableGet(CommandSpecTable *cmd_spec_table, const RedisModuleString *cmd)
+int CommandSpecTableGetFlags(CommandSpecTable *cmd_spec_table, RedisModuleDict *sub_command_tables, const RedisModuleString *cmd, const RedisModuleString *subcmd)
```

Comment:
Why not to keep `CommandSpecTableGet` and return the relevant cs?

Reply:
we can't return the cs, as my change returns a modified flags (mask out the subcommand flag).  And didn't have a use for the whole cs struct, it was only ever used for the flags from this call.

Reply:
Yes, and I wonder about `CMD_SPEC_SUBCOMMAND` too - see my other comment